README.md

```diff
@@ -1,8 +1,32 @@
-Docker Jenkins Example Setup
+# Docker Jenkins Example Setup
+
+## Introduction
+- Initialize jenkins credentials/secrets from vault.
+- Stores secrets in vault and then connects a swarm node back to the master.
+- It uses an nginx reverse proxy.
+ 
+
+## Installation
+- Fork this project to your GitHub account.
+- After forking, enter the following commands in your terminal.
+
+```
+$ git clone https://github.com/visualphoenix/docker-jenkins-demo
+$ cd docker-jenkins-demo
+```
+Next should be fairly simple to follow.
+
+## Contributing
+- The repository is open to contribution from all interested developers. Kindly send us Pull Requests with explanation as to what changes you have done.
+- Also, you can write to us by opening an [Issue](https://github.com/visualphoenix/docker-jenkins-demo/issues) and also solve a current issue if possible.
+- A detailed explanantion of how we came to building this software is maintained at the [Wiki](https://github.com/visualphoenix/docker-jenkins-demo/wiki) page.
+- This repository was created and currently maintained by @[visualphoenix](https://github.com/visualphoenix).
+
+
+##License
+
+- The software is registered under the [MIT License](https://github.com/salman-bhai/solitaire/blob/master/LICENSE)
```

Comment:
also, when you update this commit, please amend the previous one with the fixed link.